# config_iot_ros.yaml 
# IoT Configuration
config_pyiot:
  mqtt:
    server_url: "broker.mqttdashboard.com"        # http://www.hivemq.com/demos/websocket-client/
    # server_url: "test.mosquitto.org"			  # Alternative to HiveMQ
    server_port: 1883
    topic_sub: "/eyrc/vb/SwSuSukA/orders"          # <unique_id> = SwSuSukA
    topic_pub: "eyrc/SwSuSukA/ros_to_iot"          # <unique_id> = SwSuSukA
    qos: 0

    sub_cb_ros_topic: "/ros_iot_bridge/mqtt/sub"   # ROS nodes can listen to this topic to receive data from MQTT
  google_apps:
    spread_sheet_id: AKfycbylMeXYvAa6GC2mRid2qpsG_00YaCLbcFpxN83kc-vib73CzKxI   # Spreadsheet Id/Webapp Id
    
